frontend/vue/components/CodeExercise/CodeArea.vue

```diff
@@ -55,6 +56,11 @@ export default defineComponent({
         }
       }
     })
+    const test = (view: EditorView): boolean => {
```

Comment:
we should probably give this function a different name instead of `test` (maybe `keyboardRun`?)